build.sbt

```diff
@@ -1,24 +1,24 @@
-val commonsVersion     = "0.4.5"
```

Comment:
Please format the file using `scalafmt`; do not use IntelliJ for automatic formatting of scala or sbt files.